src/main/java/com/ajou/hertz/domain/user/controller/UserController.java

```diff
@@ -152,5 +153,21 @@ public UserResponse updateContactLinkV1(
 			updateContactLinkRequest.getContactLink());
 		return UserResponse.from(userUpdated);
 	}
+
+	@Operation(
+		summary = "비밀번호 변경",
+		description = "회원의 비밀번호를 변경합니다.",
+		security = @SecurityRequirement(name = "access-token")
+	)
+	@PutMapping(value = "/me/password", headers = API_VERSION_HEADER_NAME + "=" + 1)
+	public UserResponse updatePasswordV1(
+		@RequestBody @Valid UpdatePasswordRequest updatePasswordRequest,
+		@AuthenticationPrincipal UserPrincipal userPrincipal
+	) {
+		UserDto userUpdated = userCommandService.updatePassword(userPrincipal.getUserId(),
+			updatePasswordRequest.getPassword());
```

Comment:
요거 줄 정리 좀 해주시면 이쁠 거 같아요? 아니면 말구요 ㅎ;

```suggestion
		UserDto userUpdated = userCommandService.updatePassword(
		        userPrincipal.getUserId(),
			updatePasswordRequest.getPassword()
	        );
```